java-components/build-recipes-database/src/main/java/com/redhat/hacbs/recipies/build/BuildRecipeInfo.java

```diff
@@ -16,6 +16,8 @@ public class BuildRecipeInfo {
     boolean enforceVersion;
     List<String> additionalArgs = new ArrayList<>();
     List<String> ignoredArtifacts = new ArrayList<>();
+    String toolVersion;
+    String javaHome;
```

Comment:
This is not the right place for javaHome, this is something that depends on the builder image. This should just be a version and something else will resolve it into a path.

Reply:
OK, I will change `JAVA_HOME` to `JAVA_VERSION`.

Reply:
This would allow you to explicitly specify which JDK you needed to use. At the moment we just try all three until one works, when it gets a bit smarter it will try and infer which ones to use.

Reply:
If it's in the image name itself, as in the case of your images, then for `quay.io/sdouglas/hacbs-jdk8-builder:latest` I could do `"quay.io/sdouglas/hacbs-jdk" + javaVersion + "-builder:latest"` directly in the code and I don't even need it in the yaml. But with my current single image setup, I do.